# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/CookieLogging.cpp
# Commit: 232bb9649014
# Full Hash: 232bb9649014c7d8db152ca82dcd10642193814d
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2025-06-03 21:32:22
# Description:
#   Bug 1968782 - nsICookie.SAMESITE_UNSET as one of the possible cookie sameSite values for cookie logging, r=cookie-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D252221
# ==============================================================================

diff -r fa1d73a59dcd -r 232bb9649014 netwerk/cookie/CookieLogging.cpp
--- a/netwerk/cookie/CookieLogging.cpp	Tue Jun 03 12:13:25 2025 +0000
+++ b/netwerk/cookie/CookieLogging.cpp	Tue Jun 03 12:44:27 2025 +0000
@@ -22,6 +22,8 @@
       return "lax";
     case nsICookie::SAMESITE_STRICT:
       return "strict";
+    case nsICookie::SAMESITE_UNSET:
+      return "unset";
     default:
       MOZ_CRASH("Invalid nsICookie sameSite value");
       return "";
